### PR TITLE
Add additional content

### PR DESCRIPTION
**Pull request by brlin-tw**
This patch adds additional content to the template for convenience.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,12 +15,12 @@ distributions.</p>
 
 ## Install
 
-    sudo snap install my-snap-name
+    sudo snap install --channel=edge --devmode my-snap-name
```
Comment (lucyllewy) [inline README.md:18]: I think we want this to remain as-is.

File: README.md [hunk 2]
```diff
@@ -15,12 +15,12 @@ distributions.</p>
...
 
 ([Don't have snapd installed?](https://snapcraft.io/docs/core/install))
 
 <!-- Uncomment and modify this when you have a screenshot
-![my-snap-name](screenshot.png?raw=true "my-snap-name")
+![my-snap-name](snap/screenshots/screenshot.png?raw=true "my-snap-name")
```
Comment (lucyllewy) [inline README.md:23]: Files in `/snap/` *will* be copied into the package. They cannot be filtered via `stage:` or `prime:` keywords. :-1: on this one.

File: snap/patches/README.markdown [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# /snap/patches
```
Comment (lucyllewy) [inline snap/patches/README.markdown:1]: Files in /snap/ will be copied into the package. They cannot be filtered via stage: or prime: keywords. :-1: on this one.

File: snap/patches/patch-desktop-entries.sed [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+## Append '(Snappy Edition)' to the application name to make it
+## distinguishable with the other same application using different
+## software distribution technologies
+##
+## FIXME: The appended string is not localizable, the proper way to
+##        implement this is to probably use a new X-Snappy-Name keys
+##        with localestring format to let the translators fill in
+##        additional localized string and use these values to replace
+##        the Name keys here.
+s/^\(Name\(\[.\+\]\)\?=.*\)$/\1 (Snappy Edition)/g
```
Comment (lucyllewy) [inline snap/patches/patch-desktop-entries.sed:25]: I don't like calling out that an application in a snap is any different to the same application packaged elsewhere. This is an example that people _will_ use in their packages if we include it in this repo. I therefore think this line should be dropped.

File: snap/snapcraft.yaml [hunk 5]
```diff
@@ -8,8 +8,63 @@ description: |
...
+  # Patches to fix other parts
+  patches:
+    source: snap/patches
+    source-type: local
+    plugin: dump
+
+    # DISABLED: Bug #1775582 “`organize:{ /: another-dir/ }` causes the items under host root directory to be copied in another-dir” : Bugs : Snapcraft
+    # https://forum.snapcraft.io/t/organize-another-dir-causes-the-items-under-host-root-directory-to-be-copied-in-another-dir/5806
+    #organize:
+        #/: patches/
```
Comment (lucyllewy) [inline snap/snapcraft.yaml:36]: please drop the commented-out block

File: snap/snapcraft.yaml [hunk 6]
```diff
@@ -8,8 +8,63 @@ description: |
...
+
+      mkdir \
+        --parents \
+        "$SNAPCRAFT_PART_INSTALL"/patches
+      mv \
+        "$SNAPCRAFT_PART_INSTALL"/*.diff \
+        "$SNAPCRAFT_PART_INSTALL"/*.patch \
+        "$SNAPCRAFT_PART_INSTALL"/*.sed \
+        "$SNAPCRAFT_PART_INSTALL"/patches \
+        || true # Empty patches folder is allowed
```
Comment (lucyllewy) [inline snap/snapcraft.yaml:50]: this override script isn't needed, please drop.

File: snap/snapcraft.yaml [hunk 7]
```diff
@@ -8,8 +8,63 @@ description: |
...
+    stage:
+    - patches/*
+    override-prime: 'true'
```
Comment (lucyllewy) [inline snap/snapcraft.yaml:53]: replace with:

```yaml
prime: [-*]
```

File: snap/snapcraft.yaml [hunk 8]
```diff
@@ -8,8 +8,63 @@ description: |
...
+        --parents \
+        "$SNAPCRAFT_PART_INSTALL"/patches
+      mv \
+        "$SNAPCRAFT_PART_INSTALL"/*.diff \
+        "$SNAPCRAFT_PART_INSTALL"/*.patch \
+        "$SNAPCRAFT_PART_INSTALL"/*.sed \
+        "$SNAPCRAFT_PART_INSTALL"/patches \
+        || true # Empty patches folder is allowed
+    stage:
+    - patches/*
```
Comment (lucyllewy) [inline snap/snapcraft.yaml:52]: This isn't needed.

**Review by lucyllewy — CHANGES_REQUESTED** [review]
I've left a few comments on things I'd like to be changed...

File: README.md (hunk 1 above)
Comment (flexiondotorg) [reply README.md:18]: I agee. Please revert this change @Lin-Buo-Ren

File: README.md (hunk 2 above)
Comment (flexiondotorg) [reply README.md:23]: I agree. Please revert this change @Lin-Buo-Ren

File: snap/patches/README.markdown (hunk 3 above)
Comment (flexiondotorg) [reply snap/patches/README.markdown:1]: I agree. Please revert this change @Lin-Buo-Ren

File: snap/patches/patch-desktop-entries.sed (hunk 4 above)
Comment (flexiondotorg) [reply snap/patches/patch-desktop-entries.sed:25]: I agree. Also Snappy is not how we refer to Snapcraft or Snaps these days. @Lin-Buo-Ren please revert this change.

File: snap/snapcraft.yaml (hunk 5 above)
Comment (flexiondotorg) [reply snap/snapcraft.yaml:36]: I agree, please remove this @Lin-Buo-Ren. I don't think we should be adding boilerplate that is only required for a few cases.

File: snap/snapcraft.yaml (hunk 6 above)
Comment (flexiondotorg) [reply snap/snapcraft.yaml:50]: I really don't think we should be complicating a template yaml with patches. They are only required in rare cases.

File: snap/snapcraft.yaml (hunk 7 above)
Comment (flexiondotorg) [reply snap/snapcraft.yaml:53]: I'd like to see this patches stuff removed from the template yaml entirely. However, @Lin-Buo-Ren you have come up with an interesting use case. Perhaps a blog post on snapcraft.io about best practice for applying patches, when required, to a snap would be a good idea? What to collaborate on that?

**Comment by brlin-tw** [comment]
Thanks for the review, I'll find some time to drop the undesirable content.

**Comment by merlijn-sebrechts** [comment]
@Lin-Buo-Ren are you still working on this? I'll close it for the time being. Feel free to reopen it when you find time to work on it again!

**Pull request closed without merging**